lib/web3/methods/db.js

```diff
@@ -22,35 +22,45 @@
 
 var Method = require('../method');
 
-var putString = new Method({
-    name: 'putString',
-    call: 'db_putString',
-    params: 3
-});
-
-
-var getString = new Method({
-    name: 'getString',
-    call: 'db_getString',
-    params: 2
-});
-
-var putHex = new Method({
-    name: 'putHex',
-    call: 'db_putHex',
-    params: 3
-});
-
-var getHex = new Method({
-    name: 'getHex',
-    call: 'db_getHex',
-    params: 2
-});
-
-var methods = [
-    putString, getString, putHex, getHex
-];
-
-module.exports = {
-    methods: methods
+var DB = function (web3) {
```

Comment:
I agree. But it's not only up to me and I didn't want to make it a part of this pull request.